src/hooks/useTestableNetworks.ts

```diff
@@ -5,15 +5,20 @@ const getTestnet = <TTargetNetwork extends NetworkId, TTestNetwork extends Netwo
   targetNetwork: TTargetNetwork,
   testNetwork: TTestNetwork,
   currentNetwork: NetworkId,
+  secondTestNetwork?: TTestNetwork,
 ): TTargetNetwork | TTestNetwork => {
-  return currentNetwork === testNetwork ? testNetwork : targetNetwork;
+  return secondTestNetwork && currentNetwork === secondTestNetwork
+    ? secondTestNetwork
+    : currentNetwork === testNetwork
+    ? testNetwork
+    : targetNetwork;
 };
 
 export const useTestableNetworks = () => {
   const { activeChain = { id: 1 } } = useNetwork();
 
   return {
-    MAINNET: getTestnet(NetworkId.MAINNET, NetworkId.TESTNET_RINKEBY, activeChain.id),
+    MAINNET: getTestnet(NetworkId.MAINNET, NetworkId.TESTNET_GOERLI, activeChain.id, NetworkId.TESTNET_RINKEBY),
```

Comment:
would it be better to just _**kill rinkeby**_ in lieu of adding it as a `secondTestNetwork`... will we still need rinkeby?

Reply:
eh, maybe it's best to do it how you have it for now... until we're definitely ready to remove rinkeby.

Reply:
that was my thought